src/components/graphs/pagination/Pagination.less

```diff
@@ -77,31 +89,6 @@
   -webkit-box-align: center;
   -ms-flex-align: center;
   align-items: center;
-}
-
-.link-container{
-  margin: 0 4%;
-}
-
-.icon{
-  width: 2%;
-  margin: 0 auto;
-}
-
-
-.search-bar-container{
   margin: 0 6%;
 }
```

Comment:
Just want to make sure this is intended, closing bracket makes sense, but we also left that margin in there so it will now be applied to 3 selectors instead of just the search bar container

Reply:
Yeah github is folding the code weird, try opening it. .link-container, .icon and .search-bar-container each have their own margin properties here.

Reply:
ok great